docs/releasing.md

```diff
@@ -6,8 +6,68 @@ This project has [adopted Apache rules][] with necessary or pragmatic local adap
 
 Release authorization, or the delegation of release authority to one or more release engineers, should be documented by [a suitable vote][Apache Release Policy re Release Approval] of [the uPortal-home Committers][] on [uportal-dev@][].
 
+## Release steps
 
+### Finalize documentation
+
+Before cutting the release, review the [CHANGELOG][], and verify what kind of release is appropriate: major, minor, or patch (x.y.z respectively). (See [Semantic Versioning][].)
+
+### Java release
+
+Use the Maven command line client to prepare, build, sign, and publish to the
+Sonatype-hosted central open source software artifact repository.
+
+```sh
+mvn release:prepare
+mvn release:perform
+```
+
+_Iff it went well_, push the 2 new git commits to
+uPortal-Project/uportal-home:master . Both of these commits were
+generated by the mvn release commands. The first is the commit that set the
+version numbers to the version you released; the second is the commit that set
+the version numbers to the new snapshot.
+
+Assuming that your `origin` is uPortal-project/uPortal-home, you can
+push those commits using:
+
+```sh
+git push origin master
+git push origin uportal-home-maven-<version> *
+```
+* Note: This command may result with errors. You may skip this step if a new tag has been created. Run `git tag -l` to verify.
+
+If you call canonical something else (like, say, `upstream`), you'll need to
+adjust the git push commands accordingly.
+
+Then,
+
++ [Release Maven artifacts][] from
+  [central staging repository][central Maven staging repository]
+
+### Node release
```

Comment:
I don't think we typically do a Node release of uPortal-home. [It'd be `@uportal/home`](https://github.com/uPortal-Project/uportal-home/blob/42321657546e883309bf41a133caa97f47b5bc51/package.json#L2) if we did release it, but [`@uportal/home` was last released on npm 3 years ago](https://www.npmjs.com/package/@uportal/home). The project had the `private` flag set a couple years ago, it looks like [intentionally to prevent publication to npm](https://github.com/uPortal-Project/uportal-home/commit/1fd512dbce00c01765d6097330b8e29a95b11a8f).

Reply:
I do recall getting the privacy error after trying to run `npm publish`, so I didn't include that particular step. But that makes sense - I'll remove the Node release part.